dace/codegen/codegen.py

```diff
@@ -134,6 +134,9 @@ def generate_code(sdfg) -> List[CodeObject]:
         elif hasattr(node, 'map'):
             frame._dispatcher.instrumentation[node.map.instrument] = \
                 provider_mapping[node.map.instrument]
+    if hasattr(sdfg, 'instrument'):
+        frame._dispatcher.instrumentation[sdfg.instrument] = \
+            provider_mapping[sdfg.instrument]
```

Comment:
shouldn't this be just `if sdfg.instrument != dtypes.InstrumentationType.No_Instrumentation`?

Reply:
That's true, addressed.